backend/src/main/java/com/huellapositiva/application/controller/ProposalApiController.java

```diff
@@ -470,4 +471,47 @@ public void cancelProposalAsReviser(@PathVariable("id") String idProposal) {
             throw new ResponseStatusException(HttpStatus.NOT_FOUND, PROPOSAL_DOESNT_EXIST);
         }
     }
+
+    @Operation(
+            summary = "Change status of the volunteer in proposal",
+            description = "The contact person can to change the status of volunteer in a proposal to CONFIRMED/REJECTED." +
+                    "We do not send an email in MVP version, it will be added in future versions." +
+                    "This method is POST, don´t forget to use the access token as Bearer Token and use the XSRF-TOKEN, copy and paste in HEADER as X-XSRF-TOKEN." +
+                    "Steps to use this endpoint:" +
+                    "1º. Register a contact person." +
+                    "2º. Login with contact person." +
+                    "3º. Register ESAL." +
+                    "4º. Register a proposal." +
+                    "5º. Register a volunteer." +
+                    "6º. Login with volunteer." +
+                    "7º. Join volunteer in the proposal." +
+                    "8º. Login with contact person." +
+                    "9º. Use this method.",
```

Comment:
This is out of the scope of Swagger. You can describe the user journeys in the README or a wiki page

Reply:
Done